packages/@sanity/desk-tool/src/components/confirmDeleteDialog/ConfirmDeleteDialogBody.styles.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import styled from 'styled-components'
+import {Text, Card} from '@sanity/ui'
+import {ChevronDownIcon} from '@sanity/icons'
+
+export const InternalReferences = styled.ul`
+  height: 300px;
+  overflow: auto;
+  padding: 0;
+  margin: 0;
+  flex: 0 1 auto;
+
+  & > li:not(:last-child) {
+    /* margin-bottom: 1.25rem; */
+  }
+`
+
+export const Chevron = styled(ChevronDownIcon)`
+  margin-left: auto;
+  font-size: 1.25rem;
+  color: var(--card-muted-fg-color);
+`
```

Comment:
Would be better to do this with the building blocks IMO. Icons that are children of `<Text>` automatically get the right size in Sanity UI:

```tsx
<Text muted>
  <ChevronIcon />
</Text>
```